ros/src/computing/perception/localization/lib/autoware_lidar_localizer/include/autoware_lidar_localizer/util/data_structs.h

```diff
@@ -0,0 +1,409 @@
+/*
+ * Copyright 2015-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef DATA_STRUCTS_H
+#define DATA_STRUCTS_H
+
+#include <cmath>
```

Comment:
Are these data structures needed? It would be better to use the ones in `geometry_msgs`

Reply:
I do not want to depend on ROS for library as much as possible.

Reply:
Is there a reason for that decision? Since Autoware is built on ROS effectively duplicating available data type increases the maintainability needs